Source/Scene/Scene.js

```diff
@@ -1274,9 +1274,11 @@ define([
         });
 
         var newMain =
-            'void main() \n' +
-            '{ \n' +
-            '    czm_Debug_main(); \n';
+//language="glsl"
+'\
+void main()\n\
+{\n\
+    czm_Debug_main(); \n';
```

Comment:
I know it kind of breaks the uniformity of the rest of these changes, but small shader fragments like this should probably stay as string concatenations, especially when so much of the below code is building up the shader piece by piece.